app/vmselect/netstorage/tmp_blocks_file.go

```diff
@@ -65,6 +71,7 @@ func getTmpBlocksFile() *tmpBlocksFile {
 
 func putTmpBlocksFile(tbf *tmpBlocksFile) {
 	tbf.MustClose()
+	tmpBufSizeSummary.Update(float64(len(tbf.buf) / 1024))
```

Comment:
It's better to use bytes for metric - `tmpBufSizeSummary.Update(float64(len(tbf.buf))`

 It allows to properly visualize with unit hint.